Task 2 from class 6.  Family group salary

lucilakim opened the pull request.

lucilakim committed:
Modifies the directory of solved files

To organize folders and files. And separate the solved ones from the ones i am going to solve.

lucilakim committed:
Upload task 2 resolved from class 6

To see if something needs to be corrected or modified.

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+const $body = document.querySelector('body');
+const $buttonAddInput = document.querySelector('#add-input-button');
```
I think it's not necessary to refer to the HTML element ``buttonAddInput`` => ``addButton``. What do you think?

lucilakim replied:
Yes!, it looks much better that way. 
I also removed the following variable,
from $buttonRemoveInput to $buttonRemove :D

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+}
+
+$buttonRemoveInput.onclick = () => {
+    const $allLabels = document.querySelectorAll('.salary-label');
```
I think it's not necessary to add the ``all`` word in the variable, instead, you can declare ``const labels = ...`` that way you're referring to ALL labels with ``'.salary-label'`` class.

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+
+$buttonRemoveInput.onclick = () => {
+    const $allLabels = document.querySelectorAll('.salary-label');
+    const $allInputs = document.querySelectorAll('.salary-input');
```
Same goes here.

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+}
+
+$buttonCalculateSalary.onclick = () => {
+    const allWagesFromHtml = document.querySelectorAll('.salary-input');
```
I think you should keep consistency with the names you use, meaning that you should change everything to ``wage`` or ``salary``

lucilakim replied:
Yes, it's better understood this way, 
also change the id of <p id="smaller-salary"> which was lower-salary, for the same reason.

I'm going to start implementing it (insert muscle emoji)

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+
+$buttonCalculateSalary.onclick = () => {
+    const allWagesFromHtml = document.querySelectorAll('.salary-input');
+    const allWages = getNumbers(allWagesFromHtml);
```
Same here.

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+    const averageMonthlySalary = getMonthlySalary(averageSalary);
+
+    $resultContent.style.display = "block";
+    const $higerSalaryElement = document.querySelector('#higher-salary');
```
There's a typo here, ``higer`` => ``higher``

Genebson replied:
I think the ``Element`` word is not necessary, it could be ``$higherSalaryText`` or ``$higherSalary``. What do you think?

Genebson commented on introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js:
```diff
@@ -0,0 +1,130 @@
+    return false;
+}
+
+function createLabelInput(labelMessage, labelIdClass, inputIdClass) {
```
Hmmm, maybe you should split this function in two, like: ``createLabel(message, id)`` and ``createInput(id)``. What do you think?

lucilakim replied:
Much better :D!

lucilakim committed:
Correct variable names

So that it is not repetitive and there is consistency between the names of variables

lucilakim committed:
Divide function to create label and input on two

To make the function do one thing only, for the single responsibility principle:

createLabel(message, id) and createInput(id)
